section3/reflection.md

```diff
@@ -1,13 +1,18 @@
 ## Section 3 Reflection
 
-1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
+1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel? I am still getting more and more comfortable learning and trying to pull and practice information independently.  I still feel uneasy as I dont know if I am doing things correctly on my own.  I spend much more time than necessary on things right now but I think that will continue to get better as I practice the muscle memory and increase my knowledge.
 
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
+I think an application that regularly uses arrays would be netflix.  The app has peoples watching history saved and they are then able to generate lists of movies or shows that would fall in line with that particular customer. The more data the app has on what shows are being watched, the more accurate the suggestions can become.
 
 3. In your own words, explain what an Array method is.
+An array is an object that can store multiple values at once Array methods are built in functions of javascript that performs a change or calculation that can prevent a user from having to write from scratch
 
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
+The purpose of a loop is to help perform repeated tasks that have been entered without having to write the code over and over again. Using a loop in conjunction with an array can allow for you to run a loop for specific items from an extended list.
 
 5. If you had to teach a 5 year old about loops, how would you explain it?
+I would explain a loop to a 5 year old similarly to a scavenger hunt.  You have a list of all the potential objectives or tasks outlined.  You complete the tasks by following the instructions over and over by process of elimination until you are complete.
 
 6. What questions do you still have about Arrays and loops?
+My main questions about arrays and loops is just the general writing of them.  I know this is still very basic stuff in regards to writing arrays and loops but I am still struggling to comprehend how to write out variables.  I am also still having questions in regards to completing my for statement correctly. I was wasting a lot of time tinkering around trying to get the loops to perform like I was hoping.
```

Comment:
Your syntax looks good overall, I wold suggest just getting more practice between now and your cohort start date.